web-components/src/components/cards/OrderSummaryCard/OrderSummaryCard.tsx

```diff
@@ -0,0 +1,163 @@
+import { ReactNode, useState } from 'react';
+import { EditButtonIcon } from 'web-components/src/components/buttons/EditButtonIcon';
+import { DenomIconWithCurrency } from 'web-components/src/components/DenomIconWithCurrency/DenomIconWithCurrency';
+import { EditableInput } from 'web-components/src/components/inputs/new/EditableInput/EditableInput';
+import { PrefinanceTag } from 'web-components/src/components/PrefinanceTag/PrefinanceTag';
+import { SupCurrencyAndAmount } from 'web-components/src/components/SupCurrencyAndAmount/SupCurrencyAndAmount';
+import Title from 'web-components/src/components/typography/new/Title';
+
+import {
+  OrderProps,
+  OrderSummaryProps,
+  PaymentMethod,
+} from './OrderSummaryCard.types';
+
+function OrderSummmaryRowHeader({
```

Comment:
it would be nicer to put all those sub-components in dedicated files, eg OrderSummarCard.RowHeader.tsx